server/src/AMQPConnectionInfo.cc

```diff
@@ -32,6 +34,7 @@ using namespace mlpl;
 struct AMQPConnectionInfo::Impl {
 	Impl()
 	: m_URL(),
+	  m_URLBuf(NULL),
```

Comment:
OK. I see.
But we want to recognize the relationship easily.
Anyway,  I'll merge this PR.